src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -139,12 +140,21 @@ class SaveInProgressIntro extends React.Component {
     } else if (renderSignInMessage) {
       alert = renderSignInMessage(prefillEnabled);
     } else if (prefillEnabled && !verifyRequiredPrefill) {
-      const { buttonOnly, retentionPeriod, unauthStartText } = this.props;
+      const {
+        buttonOnly,
+        retentionPeriod,
+        unauthStartText,
+        unauthButtonClasses,
+      } = this.props;
+      const classes = classNames('usa-button-primary', unauthButtonClasses);
```

Comment:
can you add a screenshot of how these classes change this button's appearance?

Reply:
![Screen Shot 2020-08-18 at 11 21 05 AM](https://user-images.githubusercontent.com/1094999/90532171-eb4ded80-e144-11ea-9c29-d3eae8a1a46d.png)

For this specific form it changes the button to be `vads-u-background-color--green`

Reply:
the 10203 version vs the platform version of the component only differed by classes applied to the unauthorized button. I am removing the 10203 version and adding a property to platform version to allow for additional classes to be applied to the unauthorized button so that future changes to the SaveInProgress component are used on 10203, also it resolves the 508 issue.